Create january Product Docs release note (#5029)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/71-Mar-2024/product-docs-jan-rn.md

---
title: "January 2024 product docs updates"
id: "product-docs-jan24"
description: "Jan 2024: The Product docs team merged 110 PRs, made various updates to dbt, such as adding new Mesh FAQs, three new developer blog posts, new conversion metrics for Semantic Layer, and more."
sidebar_label: "Update: Product docs changes"
tags: [Jan-2024, product-docs]
date: 2024-03-01
sidebar_position: 09
---

> Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! First, we’d like to thank the 10 new community contributors to docs.getdbt.com :pray: What a busy start to the year! We merged 110 PRs in January.

Reviewer: I ❤️ how friendly and welcoming the opening is!

Here's how we improved the [docs.getdbt.com](http://docs.getdbt.com/) experience:

> - Added new hover behavior for images and FAQs. This was subsequently modified to improve user experience.

Reviewer: We might want to break this into two accomplishments since we walked back the hover behavior for FAQs
- Added new hover behavior for images.
- Added new expandables for FAQs

> - Added new hover behavior for images and FAQs. This was subsequently modified to improve user experience.

Author: ```suggestion
- Added new hover behavior for images
- Added new expandables for FAQs
```

- Pruned outdated notices and snippets as part of the docs site maintenance

## New content

January saw some great new content:

> - New [dbt Mesh FAQs](https://docs.getdbt.com/best-practices/how-we-mesh/mesh-4-faqs) page

Reviewer: The first bullet point above has a period at the end but none of the other ones do

- Beta launch of [Explorer’s column-level lineage](https://docs.getdbt.com/docs/collaborate/column-level-lineage) feature
- Developer blog posts:
  - [More time coding, less time waiting: Mastering defer in dbt](https://docs.getdbt.com/blog/defer-to-prod)
  - [Deprecation of dbt Server](https://docs.getdbt.com/blog/deprecation-of-dbt-server)
  - From the community: [Serverless, free-tier data stack with dlt + dbt core](https://docs.getdbt.com/blog/serverless-dlt-dbt-stack)

> - Extrica team added docs for the [dbt-extrica community adapter](https://docs.getdbt.com/docs/core/connect-data-platform/extrica-setup)

Reviewer: ```suggestion
- The Extrica team added docs for the [dbt-extrica community adapter](https://docs.getdbt.com/docs/core/connect-data-platform/extrica-setup)
```

> - Semantic Layer: New [conversion metrics docs](https://docs.getdbt.com/docs/build/conversion) and added the parameter `fill_nulls_with` to all metric types  (launching this week January 12, 2024)

Reviewer: ```suggestion
- Semantic Layer: New [conversion metrics docs](https://docs.getdbt.com/docs/build/conversion) and added the parameter `fill_nulls_with` to all metric types (launched the week of January 12, 2024).
```

> - New [dbt environment command](https://docs.getdbt.com/reference/commands/dbt-environment) and its flags for the cloud CLI

Reviewer: ```suggestion
- New [dbt environment command](https://docs.getdbt.com/reference/commands/dbt-environment) and its flags for the dbt Cloud CLI
```

## Updated content

> January also some some refreshed content, either aligning with new product features or requests from the community:

Reviewer: ```suggestion
January also saw some refreshed content, either aligning with new product features or requests from the community:
```

- Native support for [partial parsing in dbt Cloud](https://docs.getdbt.com/docs/dbt-cloud-environments#partial-parsing)
- Updated guidance on using dots or underscores in the [Best practice guide for models](https://docs.getdbt.com/best-practices/how-we-style/1-how-we-style-our-dbt-models)
- Updated [PrivateLink for VCS docs](https://docs.getdbt.com/docs/cloud/secure/vcs-privatelink)
- Added a new `job_runner` role in our [Enterprise project role permissions docs](https://docs.getdbt.com/docs/cloud/manage-access/enterprise-permissions#project-role-permissions)

> - Added saved queries to [metricflow commands](https://docs.getdbt.com/docs/build/metricflow-commands#list-saved-queries)

Reviewer: ```suggestion
- Added saved queries to [Metricflow commands](https://docs.getdbt.com/docs/build/metricflow-commands#list-saved-queries)
```
